examples/bitcoin_crate/README.md

```diff
@@ -0,0 +1,58 @@
+# Segregated Witness (SegWit) Transactions

```

Comment:
In my opinion just the example transaction below, breaking down the different fields of the transaction and illustrating the weights of the fields would be enough. We can safely assume that the user will be aware of what segwit and legacy transactions are.

Reply:
I just wanted to give a context as it's a README because when I was reading it as a standalone, it looked a bit incomplete without the first part. I have directly taken this up from learnmeabitcoin, bitcoin stack exchange and bitcoin wiki as good stepping stone articles.

Reply:
Yes. We can direct the user to links if he is interested in learning about segwit.

Reply:
Awesome. I will remove the first legacy picture, and put some references

Reply:
Done, cool